python/py-pdal/Portfile

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-pdal
+python.rootname     pdal
+version             3.2.3
+revision            0
+
+categories-append   gis
+license             BSD
+maintainers         {@nilason} openmaintainer
```

Comment:
please add you obfuscated e-mail address as well (or remove the curly brackets)

Reply:
added